tests/jerry/fail/regression-test-issue-4190-4195.js

```diff
@@ -0,0 +1,16 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default a;
+for (let
```

Comment:
As the PR fixes #4188 as well, shouldn't that ID also appear in the file name of the test (regression-test-issue-4188-4190-4195.js)?

Reply:
Indeed, I've realized later that #4188 is also fixed by the PR.